ahd2fhir/mappers/ahd_to_observation_smkstat.py

```diff
@@ -88,28 +89,28 @@ def get_smoking_status_observation_from_annotation(
     observation.category = [category]
 
     smkstat = SNOMED_LOINC_MAPPING[annotation["smokingStatus"]]
-    # valueCodeableConcept
-    value_codeable_concept = CodeableConcept()
-    value_coding = Coding.construct()
-    value_coding.system = "http://loinc.org"
-    value_coding.code = smkstat["code"]
-
-    value_coding_snomed = Coding.construct()
-    value_coding_snomed.system = "http://snomed.info/sct"
-    value_coding_snomed.code = annotation["sctid"]
-
-    value_codeable_concept.coding = [value_coding, value_coding_snomed]
-    value_codeable_concept.text = smkstat["text"]
-    observation.valueCodeableConcept = value_codeable_concept
-
-    # SNOMED concept
-    # value_codeable_concept = CodeableConcept()
-    # value_coding_snomed = Coding.construct()
-    # value_coding_snomed.system = "http://snomed.info/sct"
-    # value_coding_snomed.code = annotation["sctid"]
-    # value_codeable_concept.coding = [value_coding]
-    # value_codeable_concept.text = annotation["smokingStatus"]
-    # observation.valueCodeableConcept = value_codeable_concept
+
+    # If SMKSTAT_AS_VALUESTRING is set
+    if os.getenv("SMKSTAT_AS_VALUESTRING", "").lower() in ["true", 1, "yes"]:
```

Comment:
shouldn't this use the settings/config class instead? https://github.com/miracum/ahd2fhir/blob/master/ahd2fhir/config.py

Reply:
That sure would be the cleaner way, but as the custom mappers are hardly used by anyone but me i didn't want to bloat the Settings class.

If you think it should be there anyway i can change it.

Reply:
ah, I see. I'm OK with keeping it as-is then. I think a solution would be to encapsulate the mappers within a class that gets passed the config object in its constructor. we could refactor to that eventually.

Reply:
Sounds good, in the outlook we might want to configure the mapper behavior from outside anyway. I'll make an issue.